applications/nrf_desktop/src/hw_interface/click.c

```diff
@@ -0,0 +1,234 @@
+/*
```

Comment:
please rename the file to click_detector.c

Reply:
ok